greenwave/tests/test_api_v1.py

```diff
@@ -223,6 +223,45 @@ def test_make_decision_with_missing_required_gating_yaml(mock_results, mock_waiv
             mock_waivers.assert_called_once()
 
 
+def test_make_decision_multiple_contexts(mock_results, mock_waivers):
+    mock_results.return_value = [make_result(outcome='FAILED')]
+    mock_waivers.return_value = []
+    policies = """
+        --- !Policy
+        id: "test_policy"
+        product_versions:
+          - fedora-rawhide
+        decision_context: test_policies
+        subject_type: koji_build
+        rules:
+          - !PassingTestCaseRule {test_case_name: sometest}
+
+        --- !Policy
+        id: "test_policy_2"
+        product_versions:
+          - fedora-rawhide
+        decision_context: test_2
+        subject_type: koji_build
+        rules:
+          - !PassingTestCaseRule {test_case_name: sometest_2}
+
+        --- !Policy
+        id: "test_policy_3"
+        product_versions:
+          - fedora-rawhide
+        decision_context: test_3
+        subject_type: koji_build
+        rules:
+          - !PassingTestCaseRule {test_case_name: sometest_3}
+    """
+    response = make_decision(policies=policies, decision_context=["test_policies", "test_2"])
+    assert 200 == response.status_code
+    print(response.json)
```

Comment:
```suggestion
```

Reply:
oh yes, that would've been me debugging things. thanks!

Reply:
hmm, or I may have copied it from line 196 in `test_make_decision_with_no_tests_required_and_empty_remote_rules`. maybe that one should go too? I'll leave it alone in this PR, though, it can be done separately if necessary...